### PR TITLE
Modify ncss

### PR DESCRIPTION
**Pull request by lukyanoffpashok**

### CODE REVIEW

**Comment by lukyanoffpashok** [comment]
https://pmd.github.io/latest/pmd_java_metrics_index.html

File: aibolit/metrics/ncss/ncss.py [hunk 1]
```diff
@@ -32,19 +34,16 @@ def value(self, filename: str):
...
-                metric += 1
-            elif 'Assignment' == node_type:
-                metric += 1
-            elif 'Declaration' in node_type and 'LocalVariableDeclaration' not in node_type \
-                 and 'PackageDeclaration' not in node_type:
-                metric += 1
+        for _, destination, edge_type in dfs_labeled_edges(tree.tree, tree.root):
+            if edge_type == 'forward':
+                node_type = str(tree.get_type(destination)).split('.')[1]
+                if 'STATEMENT' in node_type and 'BLOCK_STATEMENT' not in node_type:
```
Comment (aravij) [inline aibolit/metrics/ncss/ncss.py:42]: We better list explicitly which types do we check. So we just put a set of those types and check if our type in that set.

File: aibolit/metrics/ncss/ncss.py [hunk 2]
```diff
@@ -32,19 +34,16 @@ def value(self, filename: str):
...
+                    metric += 1
+                elif node_type == 'CATCH_CLAUSE':
```
Comment (aravij) [inline aibolit/metrics/ncss/ncss.py:44]: We better compare enum items, than strings.

Comment (lukyanoffpashok) [reply aibolit/metrics/ncss/ncss.py]: ok

File: aibolit/metrics/ncss/ncss.py (hunk 1 above)
Comment (lukyanoffpashok) [reply aibolit/metrics/ncss/ncss.py]: ok

**Comment by acheshkov** [comment]
@rultor merge

**Comment by lukyanoffpashok** [comment]
@acheshkov please merge

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#475` is not assigned, can't get performer

**Pull request merged**